tests/test_trading_calendar.py

```diff
@@ -784,7 +784,7 @@ def test_session_opens_in_range(self):
             self.answers.index[0],
             self.answers.index[-1],
         )
-
+        found_opens.index.freq = None
```

Comment:
Would we prefer instead to update the answer key to include the appropriate frequency? Not sure if that's easy.

Reply:
I'm actually not convinced we want that. We're comparing our generated trading calendar (with a CustomBusinessDay frequency for that calendar) with an answer key of known good dates (but no frequency). I don't think we actually want to set the frequency of the answer key, since that's a property of the calendar logic we're testing. Newer pandas has `check_freq=False` as a parameter you pass, but that fails in pandas 18 (see https://github.com/quantopian/trading_calendars/pull/146/commits/96eb935a623af3779c2a7b4cf9b8ddf5ebc401c9). 

So we can either do a version check or set freq = None before doing the check.